iocore/net/AcceptOptions.cc

```diff
@@ -37,6 +37,7 @@ AcceptOptions::reset()
   frequent_accept       = true;
   recv_bufsize          = 0;
   send_bufsize          = 0;
+  defer_accept          = 0;
```

Comment:
Why is there a `reset()` member? It's never called, so could we just use normal constructor initialization?

Reply:
Oh, it's never called? Well, setting default value is better.